CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ## [Unreleased] - TBD
 ### Added
 - Added CEDS 0.1 x 0.1 degree emissions (in `HEMCO/CEDS/v2024-06`)
+- Added met-field dependent dust tuning factors for GCHP C24 and C30 resolution in `setCommonRunSettings.sh`.  Others to be added later.
```

Comment:
I think you can exclude C30 for now since we can benchmark with lat-lon winds and do other tests using offline dust.

Reply:
Changelog updated in [05422c1](https://github.com/geoschem/geos-chem/pull/2711/commits/05422c1aaffd0c961b5b3b94846ce55f1e86856d)